bluesky/spec_api.py

```diff
@@ -29,8 +29,10 @@
 import itertools
 from itertools import chain
 from collections import ChainMap
+from inspect import signature, Parameter
+import warnings
 
-### Factory functions for generating callbacks
+# ## Factory functions for generating callbacks
```

Comment:
Every time you laugh at anything PEP8-related, you are required to toss a quarter in the jar in @hhslepicka's office.